test_config.yaml

```diff
@@ -36,6 +36,9 @@ val_video_transcript_status_url: http://val.edx.org/video/status
 celery_worker_queue: encode_worker
 celery_deliver_queue: deliver_worker
 
+# Fernet keys
+FERNET_KEYS: ['test-current-key', 'old-key-1', 'old-key-2']
```

Comment:
Why changes in `test_config.yaml`? We are not using this test config for `test_models.py`?